pkg/provisioner/ironic/ironic.go

```diff
@@ -50,6 +51,14 @@ const (
 func init() {
 	// NOTE(dhellmann): Use Fprintf() to report errors instead of
 	// logging, because logging is not configured yet in init().
+
+	var authErr error
+	ironicAuth, inspectorAuth, authErr = clients.LoadAuth()
```

Comment:
At some point around here, or inside the clients package, we should log the auth type being used, to help with debugging later.

Reply:
No logging in `init()` :disappointed:
I really hate using `init()` we should refactor to add an initialisation method to the provisioner interface. But not now ;)

Reply:
Could we log in `main()`?